strapi/strapi
# [Typescript] - Generators

### What does it do?

Make it possible for generators to create TS files based on the nature of the project.

### How to test it?

- Go to the kitchensink-ts example in the monorepo
- Use yarn strapi generate CLI and check the generated files
